pytmx/pytmx.py

```diff
@@ -1365,15 +1367,15 @@ def iter_data(self) -> Iterable[Tuple[int, int, int]]:
     def tiles(self):
         """Yields X, Y, Image tuples for each tile in the layer.
 
-        Returns:
+        Yields:
```

Comment:
as i understand it, you can return an iterator.  if you want to change this to "Yields", then the next line needs to change, since we are not yeilding an iterator, but yielding "X, Y, Image tuples for each tile in the layer."